R/gh_fill.R

```diff
@@ -0,0 +1,14 @@
+gh_fill <- function(geohashes, precision) {
+  if (length(unique(nchar(geohashes))) > 1) {
```

Comment:
See overall comment... if we only want to allow "zooming in", we should add a check that `precision >= nchar(geohashes)`.